src/app/api.mjs

```diff
@@ -204,6 +205,26 @@ export async function createBooking(data) {
   return response.data;
 }
 
+export async function createBookingSet(data) {
+  const url = `${BOOKING_SETS_PATH}/`;
+  const response = await client.post(url, data);
+  return response.data
+}
+
+export async function deleteBookingSets({ startDate, endDate, childId, activityId }) {
+  const url = `${BOOKING_SETS_PATH}/`;
+  console.log(childId)
+  const response = await client.delete(url, {
+    params: {
+      start_date: startDate.format("YYYY-MM-DD"),
```

Comment:
Why args not in camelCase?

Reply:
becouse the BE does not convert camelCase in url params (see getBookings function)